task3.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Apple pie recipe</title>
+    </head>
+    <body>
+        <h1>Applie pie</h1>
+        <br>
+        <img src="Imagess/apple-pie.jpg" alt="Applie-piee">
+        
+
+        <h2><ins>Total Time</ins></h2>
+        <p>Prep: 20 min. Bake: 45 min.</p>
+        <br>
+        <h2><ins>Makes</ins></h2>
+        <p>8 servings</p>
+        <br>
+        <h2><ins>Ingredients</ins></h2>
+        <ol type="I">
+            <li><p><strong>1/2 cup</strong> sugar</p></li>
+            <li><p><strong>1/2 cup</strong> packed brown sugar</p></li>
+            <li><p><strong>3 tablespoons</strong> all-purpose flour</p></li>
+            <li><p><strong>1 teaspoon</strong> ground cinnamon</p></li>
+            <li><p><strong>1/4 teaspoon</strong> ground ginger</p></li>
+            <li><p><strong>1/4 teaspoon</strong> ground nutmeg</p></li>
+            <li><p><strong>6 to 7 cups</strong> thinly sliced peeled tart apples</p></li>
+            <li><p><strong>1 tablespoon</strong> lemon juice</p></li>
+            <li><p><strong>Pastry for</strong> double-crust pie (9 inches)</p></li>
+            <li><p><strong>1 tablespoon</strong> butter</p></li>
+            <li><p><strong>1 large egg</strong> white</p></li>
+            <li><p><strong>Additional</strong> sugar</p></li>
+        </ol>

```

Comment:
Predložio bih korištenje simboličke liste (`<ul>`) zato što sastojci nemaju jasno definiran redoslijed. Drugim riječima, redoslijed kojim se sastojci ispisuju nije bitan i ne utječe na informaciju koja se prenosi.

Kod uputa pak redoslijed koraka je bitan i tamo bi imalo smisla koristiti numeričku listu.